src/main/resources/arguments.json

```diff
@@ -115,6 +115,94 @@
       }
     }
   },
+  "autorole": {
+    "add*": {
+      "roles": {
+        "_type": "required",
+        "_vararg": true
```

Comment:
don't we have a different thing for this already?

Reply:
I'm not sure? This meta key isn't implemented so I'll have a look. If not, I'll implement it properly